### ISSUE TITLE
Sqlite version conflict while running rails server

### ISSUE THREAD

**Issue by gauravano** [publiclab/plots2#4767]
I just received a error - https://pastebin.com/jWq71ksp while running rails server. The highlight of the error is - `Error loading the 'sqlite3' Active Record adapter. Missing a gem it depends on? can't activate sqlite3 (~> 1.3.6), already activated sqlite3-1.4.0. Make sure all dependencies are added to Gemfile. (LoadError)
`

On a simple Google search, the error was found at https://stackoverflow.com/questions/54529739/replace-gem-error-loading-the-sqlite3-active-record-adapter-while-creating with answer and the stack overflow thread is not too old. 

Just copying the error in case others face same issue. In case, issue is faced by others, we can explicitly mention the sqlite version in the Gemfile as 

```ruby
group :sqlite, :development do
  gem 'sqlite3', '~> 1.3.6'
end
``` 

Thanks!

**Comment by gauravano** [comment]
i just saw https://github.com/publiclab/plots2/pull/4761 and may be it's due to sqlite version bump.

**Comment by gauravano** [comment]
@jywarren

**Comment by jywarren** [comment]
Hmm, uh oh... should we revert that change, do you think?

**Comment by jywarren** [comment]
I opened two possible solutions. 

https://github.com/publiclab/plots2/pull/4772 is what Gaurav suggested. 

https://github.com/publiclab/plots2/pull/4773 is an attempt to fix it at 1.4.0 and if that works, we should go with that. @gauravano @cesswairimu can you try this out?

**Comment by jywarren** [comment]
OK, merged the 1.3.6 version and closing here based on @cesswairimu's input. @cesswairimu if you wouldn't mind opening an issue for sqlite testing, that'd be helpful, thanks!
